mkposters/mkposter.py

```diff
@@ -57,14 +62,15 @@ def parse(datadir: pathlib.Path, tempdir: pathlib.Path, join_scss_file: pathlib.
     <div class="body md-typeset">
     <div class="left">
     {left_body}
-    </div>
-    <div class="right">
+    </div>""", 
+    *[f"<div class=\"Middle_{i}\">\n{body}\n</div>" for i, body in enumerate(bodies[2:-1])],
```

Comment:
Best to use `middle` rather than `Middle`, I think -- CSS has weird quirks around case sensitivity.